cassandra-analytics-integration-framework/src/main/java/org/apache/cassandra/testing/CassandraIntegrationTest.java

```diff
@@ -59,6 +59,13 @@
      */
     int numDcs() default 1;
 
+    /**
+     * This is only applied in context of multi-DC tests. Returns true if the keyspace is replicated
+     * across multiple DCs. Defaults to {@code true}
+     * @return whether the multi-DC test uses a cross-DC keyspace
+     */
+    boolean useCrossDcKeyspace() default true;
```

Comment:
It is more flexible to define keyspaces in the test directly with some helper method, instead of declaring an option in the annotation. For example, you can define keyspace with varying number of DCs and RFs per test. 
That being said. I am fine with the current code.

Reply:
Agreed. We can revisit this when we are refactoring these helpers to make them less error prone (minimize the no. params being passed).